### PR TITLE
Porting the BEPIS research machinery.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
Title. If any of you isn't aware of what the Bepis does, it's basically a money sink, sidetrack research machinery that can spit out special experimental design disks and other gadgets.

Credits to ArcaneMusic for creating this wonder, along yeeyah, XTDM, Qustinnus and Ryll-Ryll, for the nodes included in this PR (every one minus sticky duck tape at the moment).

## Why It's Good For The Game
Kevinz suggested me to port it so yea here we go.

## Changelog
:cl: Ghommie porting stuff by ArcaneMusic, yeeyah, XTDM, Qustinnus, Ryll-Ryll and probably others)
add: The BEPIS research machinery.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/pins.dm [hunk 1]
```diff
@@ -325,3 +312,18 @@
 
 /obj/item/firing_pin/security_level/pin_auth(mob/living/user)
 	return (only_lethals && !(gun.chambered?.harmful)) || ISINRANGE(GLOB.security_level, min_sec_level, max_sec_level)
+
+// Explorer Firing Pin- Prevents use on station Z-Level, so it's justifiable to give Explorers guns that don't suck.
+/obj/item/firing_pin/explorer
+	name = "outback firing pin"
+	desc = "A firing pin used by the austrailian defense force, retrofit to prevent weapon discharge on the station."
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/pins.dm:319]: What was wrong with the away pins?

File: code/modules/research/designs/weapon_designs.dm [hunk 2]
```diff
@@ -234,13 +234,13 @@
 	category = list("Firing Pins")
 	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
 
-/datum/design/pin_away
-	name = "Station Locked Pin"
-	desc = "This is a security firing pin which only authorizes users who are off station."
-	id = "pin_away"
+/datum/design/pin_explorer
+	name = "Outback Firing Pin"
+	desc = "This firing pin only shoots while ya ain't on station, fair dinkum!"
```
Comment (Trilbyspaceclone) [inline code/modules/research/designs/weapon_designs.dm:239]: This really isnt sci-y

File: code/modules/projectiles/pins.dm (hunk 1 above)
Comment (Ghommie) [reply code/modules/projectiles/pins.dm:319]: They had the same function of these ones, yet they were a little tasteless by comparison, bound to the usual RnD research.

File: code/modules/research/designs/weapon_designs.dm (hunk 2 above)
Comment (Ghommie) [reply code/modules/research/designs/weapon_designs.dm:239]: :koala: Whatever you say, fella. :kangaroo:

File: code/modules/research/techweb/nodes/bepis_nodes.dm [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+	design_ids = list("kickstart_nanites","factory_nanites","tinker_nanites","offline_nanites","synergy_nanites")
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000)
+	hidden = TRUE
+	experimental = TRUE
+
+/datum/techweb_node/interrogation
+	id = "interrogation"
+	display_name = "Enhanced Interrogation Technology"
+	description = "By cross-referencing several declassified documents from past dictatorial regimes, we were able to develop an incredibly effective interrogation device. \
+	Ethical concerns about loss of free will do not apply to criminals, according to galactic law."
```
Comment (silicons) [inline code/modules/research/techweb/nodes/bepis_nodes.dm]: please remove or change this clause to imply this is illegal.

File: code/modules/projectiles/pins.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/modules/projectiles/pins.dm:319]: I meant the name and desc -  that stuff. The "austrailian" lore is kind TG thing, didnt think we would incorporate it is all

File: code/game/machinery/hypnochair.dm [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+
+/obj/machinery/hypnochair/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = FALSE, datum/tgui/master_ui = null, datum/ui_state/state = GLOB.notcontained_state)
+	ui = SStgui.try_update_ui(user, src, ui_key, ui, force_open)
+	if(!ui)
+		ui = new(user, src, ui_key, "hypnochair", name, ui_x, ui_y, master_ui, state)
+		ui.open()
+
+/obj/machinery/hypnochair/ui_data()
+	var/list/data = list()
+	data["occupied"] = occupant ? 1 : 0
```
Comment (LetterN) [inline code/game/machinery/hypnochair.dm]:
```suggestion
	data["occupied"] = occupant ? TRUE : FALSE
```

**Pull request merged**